db/records/operations/insert.py

```diff
@@ -72,3 +73,16 @@ def insert_records_from_csv(table, engine, csv_filepath, column_names, header, d
                         temp_file.write(contents)
                     temp_file.seek(0)
                     cursor.copy_expert(copy_sql, temp_file)
+
+
+def insert_from_select(from_table, target_table, engine, mappings=None):
+    target_table_col_list = [col for col in target_table.c if col.name != 'id']
+    with engine.begin() as conn:
+        sel = select([col for col in from_table.c if col.name != 'id'])
```

Comment:
@mathemancer Is there any other way we can think of not considering the "id" column for the select statements?

Reply:
Good idea! See my above suggestion (obviously, the relevant import needs to be made).